charts/gardener/provider-local/templates/deployment.yaml

```
@@ -90,6 +90,9 @@ spec:
         {{- if .Values.gardener.version }}
         - --gardener-version={{ .Values.gardener.version }}
         {{- end }}
+        - {{- if .Values.gardener.autonomousShootCluster }}
+        - --autonomous-shoot-cluster={{ .Values.gardener.autonomousShootCluster }}
+        - {{- end }}
```

No need for empty command line parameters:

```suggestion
        {{- if .Values.gardener.autonomousShootCluster }}
        - --autonomous-shoot-cluster={{ .Values.gardener.autonomousShootCluster }}
        {{- end }}
```